src/proxy-ui-api/frontend/src/components/ui/Expandable.vue

```diff
@@ -49,15 +56,20 @@
 
 <script lang="ts">
 import Vue from 'vue';
+import { Prop } from 'vue/types/options';
 
 export default Vue.extend({
   name: 'expandable',
   components: {},
   props: {
     isOpen: {
-      type: Boolean,
+      type: Boolean as Prop<boolean>,
```

Comment:
As far as I know there is no benefit on adding the "as Prop" for basic types like boolean or string. 
At least the vscode knows the prop types without this.  
For the interfaces like Token this is useful.

Reply:
I removed the Prop but then again it would be more consistent if we just add the typescript typing to all props instead of just part of them.